.github/workflows/openshift-tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [ "1.18", "1.20", "1.22", "1.22-micro" ]
+        version: [ "1.20", "1.22", "1.22-micro" ]
```

Comment:
We should remove version specification from the rest of the matrix.

Reply:
Fixed by https://github.com/sclorg/nginx-container/pull/225/commits/42fd49a7a67c9e79625ad8259d3594ec20bc8fb6